src/xcode/ENA/ENA/Source/Utils/ENAFormatter.swift

```diff
@@ -17,6 +17,7 @@ enum ENAFormatter {
 		let dateFormatter = DateFormatter()
 		dateFormatter.dateStyle = .long
 		dateFormatter.timeStyle = .short
+		dateFormatter.dateFormat = "yyyy.MM.dd HH:mm"
```

Comment:
It will also affect other places, perhaps create a new one and use it.

Reply:
Yes, I was also thinking the same but this used only this place and should not affect others.